src/components/DataTable/DataTable.less

```diff
@@ -0,0 +1,31 @@
+.lucid-DataTable {
+	&-message-container {
+		max-width: 401px;
+		background-color: @color-white;
+		border: @border-lightBorder;
+		border-radius: @size-borderRadius;
+	}
+
+	&-message-contents {
+		padding: @size-S @size-standard @size-standard @size-standard;
+	}
+
+	&-message-title {
+		color: @color-darkGray;
+		display: block;
+		font-size: 15px;
```

Comment:
This is the second time I've seen size for title text recently, we should hoist it up into variables.less. Let's rename `@size-font-large` and its usages to `@size-font-XL` and add a variable `@size-font-L: 15px`.